members/P000603.yaml

```diff
@@ -0,0 +1,178 @@
+#US Senator Rand Paul
+bioguide: P000603
+contact_form:
+  steps:
```

Comment:
Do we need to specify driver, or is there a "default" one?  Apologies if I've asked you this before.